myapp/views.py

```
@@ -147,7 +147,7 @@ def editpost(request,id):
             postname = request.POST['postname']
             content = request.POST['content']
             category = request.POST['category']
-            
+            xyz = request.POST['category']
```

Ruff (0.8.2)

Local variable `xyz` is assigned to but never used

Remove assignment to unused variable `xyz`

(F841)